openpype/hosts/houdini/plugins/publish/validate_frame_range.py

```diff
@@ -57,7 +57,17 @@ def get_invalid(cls, instance):
             return
 
         rop_node = hou.node(instance.data["instance_node"])
-        if instance.data["frameStart"] > instance.data["frameEnd"]:
+        frame_start = instance.data.get("frameStart")
+        frame_end = instance.data.get("frameEnd")
+
+        if not (frame_start or frame_end):
```

Comment:
This is incorrect - this will enter the if statement if any of the variables is `0`.

We should either explicitly check the value is `None` or check if the value is in `instance.data`.

Some options:

```python
if any(key not in instance.data for key in ["frameStart", "frameEnd"])
```

```python
if not all(key in instance.data for key in ["frameStart", "frameEnd"])
```

```python
if not {"frameStart", "frameEnd"}.issubset(instance.data):
```

```python
required = {"frameStart", "frameEnd"}
if not required.issubset(instance.data):
```

Reply:
Thanks. I totally missed it!